symengine/cwrapper.cpp

```diff
@@ -255,6 +255,21 @@ void basic_abs(basic s, const basic a)
     s->m = SymEngine::abs(a->m);
 }
 
+void basic_sin(basic s, const basic a)
+{
+    s->m = SymEngine::sin(a->m);
+}
```

Comment:
Since there are lots of one arg functions, maybe we could use a macro here that takes in just the name of the function.